tests/frame/unpivot_test.py

```diff
@@ -64,11 +64,12 @@ def test_unpivot(
     on: str | list[str] | None,
     index: list[str] | None,
     expected: dict[str, list[float]],
-    request: pytest.FixtureRequest,
 ) -> None:
-    if on is None and index is None and "polars" not in str(constructor):
-        # TODO(2082): add support in other backends
-        request.applymarker(pytest.mark.xfail)
```

Comment:
the reason this test was failing was because of the mixed types (which I don't think we'd want to support anyway), if we make the types consistent then, with a fix in spark-like code, it passes

Reply:
🎻🚀